scripts/pretraining/run_electra.py

```diff
@@ -402,24 +403,28 @@ def train(args):
             log_rtd_loss += sum([ele.as_in_ctx(ctx_l[0])
                                  for ele in rtd_loss_l]).asnumpy()
             log_total_loss += sum([ele.as_in_ctx(ctx_l[0])
-                                   for ele in loss_l]).asnumpy() * loss_denom
+                                   for ele in loss_l]).asnumpy()
```

Comment:
Does `log_total_loss` need to multiply by this coefficient `args.num_accumulated` here?

Reply:
I think each loss was divided by `args.num_accumulated` so there is no need to multiply `args.num_accumulated`

Reply:
@sxjscience `log_total_loss` is only used in logging.info(). And I notice that the logging phase also has some problems (but irrelevant to the training), I will fix this later.

Reply:
During the log recording phase, is `log_total_loss` equal to the weighted sum of `log_rtd_loss` and `log_mlm_loss` for now?